streams/src/test/java/org/apache/kafka/streams/integration/StreamsUncaughtExceptionHandlerIntegrationTest.java

```diff
@@ -136,30 +136,9 @@ public void teardown() throws IOException {
         purgeLocalStreamsState(properties);
     }
 
-    @Test
-    public void shouldShutdownThreadUsingOldHandler() throws Exception {
```

Comment:
Given that we remove all test for the old handler, we should remove the class level annotation to suppress deprecation warnings, too.